po/de.po

```diff
@@ -113,7 +113,7 @@ msgstr "Adresse eingetragen."
 
 #: alias.c:394
 msgid "Error seeking in alias file"
-msgstr "Fehler beim Suchen in alias Datei"
+msgstr "Fehler beim Suchen in Addressbuchdatei"
```

Comment:
Aliases in mutt are not just short mnemonics for longer strings. You can use the alias database just like a real address book - when (for example) composing mail, hit `<TAB>` when asked to provide a the recipient address and you get a list of all entries in the alias file that you can now search by whatever you want.

I'm also not sure what a better translation would be.